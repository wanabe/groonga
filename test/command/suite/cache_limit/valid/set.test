cache_limit --max 1000
cache_limit


