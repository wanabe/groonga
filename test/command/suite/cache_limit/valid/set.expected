cache_limit --max 1000
[[0,0.0,0.0],100]
cache_limit
[[0,0.0,0.0],1000]
